lib/index.ts

```diff
@@ -191,12 +192,13 @@ class PWMetrics {
       console.log(messages.getMessage('LAUNCHING_CHROME'));
       this.launcher = await launch({
         port: this.flags.port,
-        chromeFlags: parseChromeFlags(this.flags.chromeFlags),
+        chromeFlags: this.parsedChromeFlags,
         chromePath: this.flags.chromePath
       });
       this.flags.port = this.launcher.port;
       return this.launcher;
     } catch(error) {
+      console.error(error);
```

Comment:
In my case, I was installing `Lighthouse` globally before `pwmetrics`.
The original error
```
TypeError: flags.trim is not a function
    at parseChromeFlags (/Users/vinhlh/.nvm/versions/node/v8.9.4/lib/node_modules/pwmetrics/node_modules/lighthouse/lighthouse-cli/run.js:33:13)
    at PWMetrics.<anonymous> (/Users/vinhlh/.nvm/versions/node/v8.9.4/lib/node_modules/pwmetrics/lib/index.js:182:34)
    at Generator.next (<anonymous>)
    at /Users/vinhlh/.nvm/versions/node/v8.9.4/lib/node_modules/pwmetrics/lib/index.js:9:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/vinhlh/.nvm/versions/node/v8.9.4/lib/node_modules/pwmetrics/lib/index.js:5:12)
    at PWMetrics.launchChrome (/Users/vinhlh/.nvm/versions/node/v8.9.4/lib/node_modules/pwmetrics/lib/index.js:175:16)
    at PWMetrics.<anonymous> (/Users/vinhlh/.nvm/versions/node/v8.9.4/lib/node_modules/pwmetrics/lib/index.js:111:28)
    at Generator.next (<anonymous>)
    at /Users/vinhlh/.nvm/versions/node/v8.9.4/lib/node_modules/pwmetrics/lib/index.js:9:71
```